src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/ListObjectTypeVisitor.cs

```diff
@@ -16,6 +16,8 @@ namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Visitors
     /// </summary>
     public class ListObjectTypeVisitor : TypeVisitor
     {
+        private readonly Regex _schemeKeyRegex = new Regex("(?<num>[0-9])+$");
```

Comment:
Because the regular expression instance is quite expensive, it would be great to avoid it, as much as we can. Is there any other way to achieve without using the RegEx instance?